### ISSUE TITLE
Manage console - uncaught TypeError causes white screen

### ISSUE THREAD

**Issue by czomo** [dragonflyoss/Dragonfly2#1322]
### Bug report:

Manage console is broken in `configuration/scheduler-cluster` path. I can't see the details of scheduler cluster. See the screenshot for more details.
![Screenshot from 2022-05-23 12-55-41](https://user-images.githubusercontent.com/35335044/169806124-b3b795a8-01a0-40a2-98b1-0815d1a8440a.png)

### Expected behavior:

We should be able to see scheduler cluster dashboard in manage console.
### How to reproduce it:

Deploy Dragonfly with helm charts using below configuration with https://github.com/dragonflyoss/helm-charts/pull/74 patch
```
containerRuntime:
  containerd:
    enable: true
manager:
  ingress:
    enable: true
    className: private
    hosts:
      - "dragonfly.example.com"
    tls:
      - secretName: secure-tls
        hosts:
          - "dragonfly.example.com"
cdn:
  # -- Enable cdn
  enable: true
```

Enter dragonfly.example.com after ingress is up. Login to console. Go to `http://dragonfly.example.com/configuration/scheduler-cluster`. As the result we see the dashboard for a moment and then it turn into whitescreen. 

### Environment:
- Dragonfly version: v2.0.3-beta.3
- OS: Ubuntu 20.04.3 LTS
- Kernel (e.g. `uname -a`): 5.11.0-1021-aws
- Others: containerRuntime: containerd://1.4.12

**Comment by gaius-qi** [comment]
We'll fix it this week.

**Comment by czomo** [comment]
I see that it has been fixed in v2.0.3. Cool, I am closing it
